EIPS/eip-7002.md

```diff
@@ -63,7 +63,7 @@ with type `0x01` and consists of the following fields:
 3. `amount`: `uint64`
 
 The [EIP-7685](./eip-7685.md) encoding of a withdrawal request is computed as follows.
-Note that `amount` is returned by the contract little-endian, and must be encoded as such.
+Note that `amount` is stored as big-endian in storage, but is returned by the contract little-endian, and must be encoded as such.
```

Comment:
Can we clarify the "returned by" here? What is the "returned" here? (this is the log part, right? We cannot query this directly as via an `eth_call`?)

Reply:
I checked the [eas code](https://github.com/ethereum/sys-asm/blob/620ca017b279071de3ed5aeef36217d547239dd9/src/withdrawals/main.eas) and it seems like you cannot read this information by calling the contrat.

If you are not the system address, calling the contract just returns the fee. 

So I really don't know what return means here.